linera-base/src/crypto.rs

```diff
@@ -430,6 +430,12 @@ impl CryptoHash {
     pub fn as_bytes(&self) -> &HasherOutput {
         &self.0
     }
+
+    /// Returns the hash of `TestSignable(s)`, for testing purposes.
+    #[cfg(any(test, feature = "test"))]
+    pub fn for_testing(s: impl Into<String>) -> Self {
```

Comment:
`test_hash`

Reply:
That may read as a verb :/ I still think `CryptoHash::for_testing` reads better than `CryptoHash::test_hash`.

Reply:
As discussed, I used `test_hash` for now; happy to change it in a later PR if we find something better.